src/session/SessionMiddleware.ts

```diff
@@ -58,17 +58,15 @@ const sessionRequestHandler = (config: CookieConfig, sessionStore: SessionStore,
 
         onHeaders(response, () => {
             if (request.session) {
```

Comment:
Just curious, what is the `if (request.session)` statement checking, and if not true, is it correct that we don't want to refresh the cookie?

Reply:
Presumably it checks if a session is present, if not then there is no point in having a cookie to store it's id. Maybe for the initial splash pages of services.